ibis/pyspark/timecontext.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional
+
+import pyspark.sql.functions as F
+
+import ibis.common.exceptions as com
+from ibis.expr.timecontext import TIME_COL
+from ibis.expr.typing import TimeContext
+
+
+def filter_by_time_context(df, timecontext: Optional[TimeContext] = None):
```

Comment:
Can you type this

Reply:
+1. Typed with DataFrame

Reply:
Sorry what do you mean by "Typed with DataFrame"? I didn't see the typing on df.

Reply:
I just @ you again in line of those changes, maybe you were looking at an outdate version?